src/offset/local/unix.rs

```diff
@@ -11,10 +11,18 @@
 use std::{cell::RefCell, collections::hash_map, env, fs, hash::Hasher, time::SystemTime};
 
 use super::tz_info::TimeZone;
-use super::{DateTime, FixedOffset, Local, NaiveDateTime};
+use super::{FixedOffset, NaiveDateTime};
 use crate::{Datelike, LocalResult};
 
-pub(super) fn naive_to_local(d: &NaiveDateTime, local: bool) -> LocalResult<DateTime<Local>> {
+pub(super) fn offset_from_utc_datetime(utc: &NaiveDateTime) -> LocalResult<FixedOffset> {
+    get_offset(utc, false)
+}
+
+pub(super) fn offset_from_local_datetime(local: &NaiveDateTime) -> LocalResult<FixedOffset> {
+    get_offset(local, true)
+}
+
+fn get_offset(d: &NaiveDateTime, local: bool) -> LocalResult<FixedOffset> {
```

Comment:
`get_` prefixes are mostly considered unidiomatic in Rust, just call this `offset()`?